src/main.rs

```diff
@@ -41,20 +42,18 @@ async fn main() -> std::io::Result<()> {
     });
 
     HttpServer::new(move || {
+        let cors = Cors::default()
+            .allow_any_origin()
+            .allow_any_method()
+            .allow_any_header();
         App::new()
             .app_data(web::Data::new(pool.clone()))
             .app_data(web::Data::new(manager_handle.clone()))
             .wrap(Logger::default())
-            .service(
-                web::scope("/api")
-                    .wrap(AuthMiddleware)
-                    .configure(routes::init),
-            )
-            .service(
-                web::resource("/ws")
-                    .wrap(AuthMiddleware)
-                    .route(web::get().to(websocket_handler)),
-            )
+            .wrap(AuthMiddleware)
+            .wrap(cors)
+            .service(web::scope("/api").configure(routes::init))
+            .service(web::resource("/ws").route(web::get().to(websocket_handler)))
     })
     .bind("127.0.0.1:4925")?
```

Comment:
Good one here, will it be okay to get the bind value from env variables?

Reply:
Yeah, I can do that